src/components/drawer/ComponentDetailPanel.vue

```diff
@@ -102,22 +147,49 @@ function getAttribute(component, definition) {
 }
 
 /**
- * Get an array of attributes corresponding the definedAttributes array.
- * @param {Object} component - Component containing the available attribute and all definitions.
- * @return {Array} an array of attributes.
+ * Get an array of attributes corresponding to the definedAttributes array.
+ * @param {Object} component - Component containing the referenced attribute and all definitions.
+ * @return {Array} an array of referenced attributes.
  */
-function getAttributes(component) {
+function getReferencedAttributes(component) {
   return component.definition.definedAttributes.map((defAttr) => getAttribute(component, defAttr));
 }
 
+/**
+ * Get an array of attributes corresponding to the attributes without definition.
+ * @param {Object} component - Component containing the unreferenced attribute.
```

Comment:
`Component containing the available attributes`

Reply:
done